### PR TITLE
Extending runtime dependency on Stripe gem from version 5 through 11

### LINKED ISSUE #847: Adding support for Stripe gem version 7
It would great to extend runtime dependency on Stripe gem from version 6 to 7.

Runtime_dependencies

### PR DESCRIPTION
**Pull request by smakani**
Closes #847

### CODE REVIEW

**Comment by smakani** [comment]
This PR fixes issue #847. Please upvote to get it merged.

**Comment by courtsimas** [comment]
how about through 8, now.

**Comment by barsbek** [comment]
Extending to the latest version would be really helpful

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ version `3.0.0` has [breaking changes](https://github.com/stripe-ruby-mock/strip
 ### Requirements
 
 * ruby >= 2.4.0
-* stripe >= 5.0.0
+* stripe >5 & <=7.1.0
```
Comment (alexmamonchik) [inline README.md:33]:
```suggestion
* stripe > 5 & < 11
```

File: stripe-ruby-mock.gemspec [hunk 2]
```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'stripe', '> 5', '< 6'
+  gem.add_dependency 'stripe', '> 5', '<= 7.1.0'
```
Comment (alexmamonchik) [inline stripe-ruby-mock.gemspec:25]:
```suggestion
  gem.add_dependency 'stripe', '> 5', '< 11'
```

**Review by alexmamonchik — APPROVED** [review]

**Comment by alexmamonchik** [comment]
@smakani please, solve conflicts and use `init-v4` branch. We'll collect all required fixes for v4 and newest Stripe. Then merge it to master and release

**Pull request merged**